packages/@aws-cdk/aws-appsync/lib/graphqlapi.ts

```diff
@@ -292,6 +305,16 @@ export interface GraphqlApiProps {
    * @default - false
    */
   readonly xrayEnabled?: boolean;
+
+  /**
+   * The domain name configuration for the GraphQL API
+   *
+   * The hosted zone and CName must be configured in addition to this setting to
+   * enable custom domain URL
+   *
+   * @default - none a unique name is generated
```

Comment:
Can you add a comma or parenthesis after "none" ?

Reply:
+1